libchisel/src/lib.rs

```diff
@@ -34,11 +41,13 @@ pub enum ModuleError {
 }
 
 /// Utility interface for chisel modules.
-pub trait ChiselModule<'a>: Sized {
+pub trait ChiselModule<'a> {
```

Comment:
Any reason to remove sized?

Reply:
Not needed, and stops me from casting between traits.